lib/worker-pipeline.js

```diff
@@ -69,6 +70,26 @@ class AssetComputeWorkerPipeline {
         return engine;
     }
 
+    normalizeInputOuput(input, output) {
+        // TODO: we will have special cases for beta-worker-creative 
+        // special case for sensei
+        if(output.fmt === 'machine-metadata-json') {
+            output.type = 'machine-metadata-json';
+        } else {
+            // rendition.fmt should always exist
+            const mimetype = mime.lookup(output.fmt.toLowerCase());
+            output.type = mimetype && mimetype.toLowerCase();
+        }
+
+        // source.mimetype will always exist, no need to cultivate here.
+        input.type = input.mimetype && input.mimetype.toLowerCase();
+
+        if(!input.type || !output.type) {
+            //TODO should throw ClientError ?
+            console.log("Pipeline won't be able to find a plan and will throw error");
```

Comment:
For MVP, not having a mimetype would be OK, given the types we support should be found.
So You could throw here.

Longer term: We'll need a fallback to find out mimetypes, possibly using `file`+`imagemagick` (one alone does not cover all input types we claim to support). Feel free to write a JIRA story for this if you feel like it :).